
package org.fundaciobit.projectebase.archetype.ws.api.v1;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.2.12-patch-04
 * Fri Jul 04 08:48:46 CEST 2014
 * Generated source version: 2.2.12-patch-04
 * 
 */

@WebFault(name = "WsI18NError", targetNamespace = "http://impl.v1.ws.archetype.projectebase.fundaciobit.org/")
public class WsI18NException extends Exception {
    public static final long serialVersionUID = 20140704084846L;
    
    private org.fundaciobit.projectebase.archetype.ws.api.v1.WsI18NError wsI18NError;

    public WsI18NException() {
        super();
    }
    
    public WsI18NException(String message) {
        super(message);
    }
    
    public WsI18NException(String message, Throwable cause) {
        super(message, cause);
    }

    public WsI18NException(String message, org.fundaciobit.projectebase.archetype.ws.api.v1.WsI18NError wsI18NError) {
        super(message);
        this.wsI18NError = wsI18NError;
    }

    public WsI18NException(String message, org.fundaciobit.projectebase.archetype.ws.api.v1.WsI18NError wsI18NError, Throwable cause) {
        super(message, cause);
        this.wsI18NError = wsI18NError;
    }

    public org.fundaciobit.projectebase.archetype.ws.api.v1.WsI18NError getFaultInfo() {
        return this.wsI18NError;
    }
}
